src/pkg/cli/agree_tos.go

```diff
@@ -26,11 +27,12 @@ func InteractiveAgreeToS(ctx context.Context, c client.FabricClient) error {
 	err := survey.AskOne(&survey.Confirm{
 		Message: "Do you agree to the Defang terms of service?",
 		Help:    "You must agree to the Defang terms of service to continue using this tool",
-	}, &agreeToS)
+	}, &agreeToS, survey.WithStdio(term.DefaultTerm.Stdio()))
```

Comment:
I see us passing `survey.WithStdio(term.DefaultTerm.Stdio())` to `survey.AskOne` eight times. I think it's time to factor out a helper function. I would create our own `survey` pkg and export `AskOne` which would delegate to the 3rd party `survey` pkg with `survey.WithStdio(term.DefaultTerm.Stdio())` as the first `AskOpt`.

Reply:
There's also #95